src/state/contexts/GlobalState.tsx

```diff
@@ -89,15 +89,15 @@ export function GlobalStateProvider({
   const [updatingTicker, setUpdatingTicker] = useState(false);
 
   useEffect(() => {
-    if (state.ioTicker === initialState.ioTicker && !updatingTicker) {
+    if (!updatingTicker) {
       updateTicker();
     }
   }, []);
 
   async function updateTicker() {
     try {
       setUpdatingTicker(true);
-      const ticker = 'tIO'; // TODO, use contract to get ticker
+      const ticker = (await state.arioContract.getInfo()).Ticker;
       dispatchGlobalState({ type: 'setIoTicker', payload: ticker });
     } catch (error) {
       console.error(error);
```

Comment:
might be worth adding a backup here to set to "ARIO"

Reply:
[040afc3](https://github.com/ar-io/arns-react/pull/615/commits/040afc3e6a79e21822c6ca030ab167a64001f4b4)